microlith/prometheus/alerting/couchbase/couchbase-rules.yaml

```diff
@@ -37,3 +37,22 @@ groups:
         description: The percentage of time that the virtual CPU on node {{ $labels.instance }} is waiting for a real CPU is above 3%.
           This can be due to the fact that a virtual Machine is undersized or the hypervisor is overcommited.
         remediation: Increase the resources available to the VM or reduce the number of VMs on the host machine.
+
+    - alert: CB90043-diskCommitFail
+      expr: |
+        increase(kv_ep_data_write_failed[1m]) > 0
+      for: 0m
+      labels:
+        job: couchbase_prometheus
+        kind: bucket
+        health_check_id: CB90043
+        health_check_name: diskCommitFail
+        cluster: '{{ $labels.cluster }}'
+        node: '{{ $labels.instance }}'
+        bucket: '{{ $labels.bucket }}'
+        severity: critical
+      annotations:
+        title: "Disk Commit Fail on bucket: {{ $labels.bucket }}"
+        description: A disk commit has failed on {{ $labels.bucket }} (cluster {{ $labels.cluster }}, node {{ $labels.hostname }}).
+        remediation: Review your infrastructure for signs of disk problems or any other issues, or contact Couchbase Technical Support.
+        
```

Comment:
You've left a single blank space on this line, which means it's not *technically* an empty line

Reply:
😁  darn